source/elements/oneTBB/source/flow_graph/functional_node_policies.rst

```diff
@@ -56,6 +56,10 @@ with the behavior defined by the ``lightweight`` policy. For example, if the def
 ``Policy`` is ``queueing``, specifying ``lightweight`` as the ``Policy`` value is equivalent to
 specifying ``queueing_lightweight``.
 
+The ``lightweight`` policy requires a function call ``operator()`` of a node's
+body be ``noexcept``. If it is not the case, specifying ``lightweight`` policy
+to the node has no effect.
+
```

Comment:
It's true not just ``lightweight`` but also for combined policies, right? I would rephrase to something like "The function call operator of a node body must be ``noexcept`` for lightweight policies to have effect".

Also I think the "hint" nature of the lightweight policies should be better described in the specification; "helps to reduce the overhead" is very vague. This improved general description would also mention that the effect is implementation-specific and "best effort", completely ignoring the lightweight hint is valid. Then we can describe ``noexcept`` as the necessary condition (like I did above) without additionally saying that there is no effect if the condition is not met (because "no effect" is always a valid implementation option).